pkg/remote/remote.go

```diff
@@ -573,3 +579,13 @@ func GetOriginalCWD(workingDirectoryCtrl filesystem.WorkingDirectoryInterface, m
 	manifestPathDir := filepath.Dir(filepath.Clean(fmt.Sprintf("/%s", manifestPath)))
 	return strings.TrimSuffix(cwd, manifestPathDir), nil
 }
+
+func getOktetoPrefixEnvVars(environ []string) map[string]string {
+	prefixEnvVars := make(map[string]string)
+	for _, v := range environ {
+		if strings.HasPrefix(v, "OKTETO_") {
```

Comment:
it's unlikely to happen if we use `os.Environ()` as the source, but you never know. This func will panic for values such as `"OKTETO_ENV_VAR_2`, with missing `=`.

We had a similar regression lately (see https://github.com/okteto/okteto/pull/4426/files), I would write defensive code, ensuring the edge-cases are covered, just in case.